app/controllers/bids_controller.rb

```diff
@@ -1,18 +1,23 @@
 class BidsController < ApplicationController
 
   def create
-    @bid = Bid.new(params.require(:bid).permit(:price, :house_id))
-    @bid.user_id = @current_user.id
-    if @bid.house_id.nil?
+    @bid = @current_user.bids.new(params.require(:bid).permit(:price, :house_id))
+    @house = House.find_by(zillow_id: params[:house][:zillow_id])
+    @old_bid =@house.bids.last
```

Comment:
I'm getting an error here if the house has not been created yet. I think this should move after the house is found or created.